assets/shaders/irradiance_volume_voxel_visualization.wgsl

```diff
@@ -0,0 +1,33 @@
+#import bevy_pbr::forward_io::VertexOutput
+#import bevy_pbr::irradiance_volume
+#import bevy_pbr::mesh_view_bindings
+
+struct VoxelVisualizationIrradianceVolumeInfo {
+    transform: mat4x4<f32>,
+    inverse_transform: mat4x4<f32>,
+    resolution: vec3<u32>,
+    intensity: f32,
+}
+
+@group(2) @binding(100)
+var<uniform> irradiance_volume_info: VoxelVisualizationIrradianceVolumeInfo;
+
+@fragment
+fn fragment(mesh: VertexOutput) -> @location(0) vec4<f32> {
```

Comment:
Why do the cubes appear to be gray+shaded? I was thinking we would want to show essentially an "unlit" solid-color cube the color of the probe.

Reply:
They are doing what you suggested, I'm fairly sure. I don't see any shading.

I believe the gray comes from the grayish background color, which is roughly the same in Blender so the indirect light from it is captured in the probe. The sides of the cubes will be a mix of the gray from the background with the colors of the floor visible from that cube side. The relative mix of gray vs. floor color is based on how much that side of the cube can "see".

Reply:
So the irradiance volumes are sampling the clear color/background? Seems strange. ClearColor shouldn't be used for lighting. Only a Skybox if specified, but not the clear color itself.

Reply:
In reality it's sampling the "world" in Blender which happens to be dark gray by default. Would you prefer that I change it to black?

Reply:
Yeah I think that would be better. Or drop in a skybox and use that.

Reply:
OK, I added a skybox and re-baked the irradiance volume. It looks right to my eyes.